part2/stages/Functions/install-main

```diff
@@ -540,35 +540,48 @@ install_bootloader_from_dom0fs()
     return 0
 }
 
+remove_efi_boot_entries()
+{
+    echo "Removing any existing OpenXT bootloader entries" >&2
+    for entry in $( efibootmgr | awk '/ OpenXT/ { print $1 }' ); do
```

Comment:
I guess this is more robust then running `do_cmd efibootmgr -D >&2` we already do at another point?

Reply:
I put this in the PR text, but not the commit message: "The existing call to efibootmgr -D removes duplicates from the BootOrder Variable - not duplicate entries."  `-D` doesn't remove identical `Boot0001* OpenTX` and `Boot0002* OpenXT` entries.  It would only remove duplicate Boot000X entries from BootOrder.

Reply:
Ah, got it! SGTM